src/ploomber/tasks/notebook.py

```diff
@@ -757,23 +801,33 @@ def run(self):
         _write_text_utf_8(tmp, self.source.nb_str_rendered)
 
         if self.local_execution:
-            self.papermill_params["cwd"] = str(self.source.loc.parent)
+            self.nb_executor_params["cwd"] = str(self.source.loc.parent)
 
         # use our custom engine
         if self.debug_mode == "now":
-            self.papermill_params["engine_name"] = "debug"
+            self.nb_executor_params["engine_name"] = "debug"
         elif self.debug_mode == "later":
-            self.papermill_params["engine_name"] = "debuglater"
-            self.papermill_params["path_to_dump"] = f"{self.name}.dump"
+            self.nb_executor_params["engine_name"] = "debuglater"
+            self.nb_executor_params["path_to_dump"] = f"{self.name}.dump"
 
         # create parent folders if they don't exist
         Path(path_to_out_ipynb).parent.mkdir(parents=True, exist_ok=True)
 
         try:
             # no need to pass parameters, they are already there
-            pm.execute_notebook(
-                str(tmp), str(path_to_out_ipynb), **self.papermill_params
-            )
+            if self.engine == "ploomber_engine":
```

Comment:
this should be ploomber-engine

```suggestion
            if self.engine == "ploomber-engine":
```